eclipse/plugin.xml

```diff
@@ -514,7 +514,7 @@
       <command
             categoryId="org.eclipse.ui.category.project"
             id="saros.ui.commands.ShareResource"
-            name="Share Project(s)...">
+            name="Share Resource(s)...">
```

Comment:
Resource is an internal Eclipse specific API object.  I do not know if this is the best name we should use.

Reply:
What else should we call it?

I was first going to call it "Share Reference Points" (and was going to name the negotiation "reference point negotiation", but Kelvin pointed out that "sharing resources" is an easy concept for the user to understand, so I decided to use it.

Reply:
Are there any plans for testing the usability changes of your patches ? If this is the case you can stick with Resource for now and wait for any issues that might arise with the new sharing logic.

Reply:
> Are there any plans for testing the usability changes of your patches ? If this is the case you can stick with Resource for now and wait for any issues that might arise with the new sharing logic.

I did not have any further usability test planned, no. But this should not be too hard to change later on.

As already mentioned, I would like to mainly concentrate on writing the actual thesis document for now (and getting the existing code reviewed). I already expanded the scope of the thesis, but I have to draw the line at some point. (I am still working with a deadline.)